Appraisals

```diff
@@ -1,34 +1,6 @@
 # frozen_string_literal: true
 
-appraise "rubocop-0.60" do
-  gem "rubocop", "~> 0.60"
-  gem "rails", "~> 5.2"
-end
-appraise "rubocop-0.61" do
-  gem "rubocop", "~> 0.61"
-  gem "rails"
-end
-appraise "rubocop-0.62" do
-  gem "rubocop", "~> 0.62"
-  gem "rails"
-end
-appraise "rubocop-0.63" do
-  gem "rubocop", "~> 0.63"
-  gem "rails"
-end
-appraise "rubocop-0.64" do
-  gem "rubocop", "~> 0.64"
-  gem "rails"
-end
-appraise "rubocop-0.65" do
-  gem "rubocop", "~> 0.65"
-  gem "rails"
-end
-appraise "rubocop-0.66" do
-  gem "rubocop", "~> 0.66"
-  gem "rails"
-end
-appraise "rubocop-0.67" do
-  gem "rubocop", "~> 0.67"
```

Comment:
If these are dropped, then the files from `gemfiles/rubocop_0.60.gemfile` to `gemfiles/rubocop_0.67.gemfile` will also need to be removed.

Reply:
Done!